addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -127,6 +127,31 @@ if (_platform in ["linux", "osx"]) then {
 };
 
 
+
+if (hasInterface && {_platform == "windows"}) then {
```

Comment:
Is there a reason why it should check for `hasInterface`?

Reply:
We don't use extensions on headless server/clients
we might want to remove the version check above as well for them?

Reply:
For consistency's sake, we should remove the version check then.